learn/function-calling/project_using_firefunction_v1/.gitignore

```diff
@@ -0,0 +1,6 @@
+/fc
```

Comment:
Where did you copy `project_using_firefunction_v1` from?

Reply:
I copied it from here: https://github.com/fw-ai/ai-starter-kits/tree/main/function-calling/firefunction

It's supposed to be the code for this blog post: https://fireworks.ai/blog/function-call-vercel-fastapi-serp
But some of the the links are broken in the blog post, so it took a bit of digging to confirm. 

The project doesn't look like it was fully-setup to be run locally - so I'm not 100% what would be the best approach. 

option 1
- Add a ReadME (that has the salient info from the blog posts) 
- Adapt the instructions for running the project in a virtual env locally
Then leave it add that

option 2
???

Reply:
option 2 sgtm